web-ui/src/main/resources/catalog/templates/editor/directory.html

```diff
@@ -184,39 +184,37 @@ <h5 class="btn-group">
                           {{'directoryEntryValidate' | translate}}</a>
                         </li>
                         <li><a href ng-click="rejectEntry(e)">
-                          <i class="fa fa-ban fa-fw" />&nbsp;
+                          <i class="fa fa-ban fa-fw text-danger" />&nbsp;
                           {{'directoryEntryReject' | translate}}</a>
                         </li>
                         <li><a href ng-click="delEntry(e)">
-                          <i class="fa fa-times fa-fw text-danger" />&nbsp;
+                          <i class="fa fa-trash fa-fw" />&nbsp;
                           {{'delete' | translate}}</a>
                         </li>
                       </ul>
                     </div>
 
                     <div class="btn-group">
                       <button class="btn btn-default btn-sm dropdown-toggle"
+                        ng-click="e.openAssociatedMD = true"
```

Comment:
The associated records are retrieved using a search on xlink field (eg. http://localhost:8080/geonetwork/srv/fre/q?_content_type=json&_isTemplate=y+or+n&_xlink=*local:%2F%2Fsrv%2Fapi%2Fregistries%2Fentries%2F9b0bd8f9-ee1c-4919-92ee-5ee9229d5195*&any=&bucket=metadata&fast=index&from=1&sortBy=title&to=8). This menu should not be displayed if XLink resolution is not activated (see https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/edit/directoryentryselector/partials/directoryentryselector.html#L49)